Libraries/MiscDrivers/ExtMemory/is25.c

```diff
@@ -0,0 +1,616 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2023 Analog Devices, Inc. All Rights Reserved. This software is
+ * proprietary and confidential to Analog Devices, Inc. and its licensors.
+ *
+ ******************************************************************************/
```

Comment:
Please use the updated copyright in the newly added files: 
```
/******************************************************************************
 *
 * Copyright (C) 2022-2023 Maxim Integrated Products, Inc. All Rights Reserved.
 * (now owned by Analog Devices, Inc.),
 * Copyright (C) 2023 Analog Devices, Inc. All Rights Reserved. This software
 * is proprietary to Analog Devices, Inc. and its licensors.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *
 ******************************************************************************/
```

The main branch already has the updated copyrights in all the existing files. In theory, you will just have to update the copyrights in the newly added files. The existing files should have their copyrights automatically updated (assuming no changes were made to it) when the final `dev/me13b_me55-examples` branch is merged into main.

Reply:
Updated.